composer.json

```diff
@@ -39,15 +39,21 @@
 
         "sonata-project/cache-bundle": "^2.1",
         "sonata-project/doctrine-phpcr-admin-bundle": "^2.0",
-        "sonata-project/admin-bundle": "^3.2",
+        "sonata-project/admin-bundle": "3.2 as 3.6",
         "sonata-project/translation-bundle": "^2.0",
 
         "jms/serializer-bundle": "^1.1",
         "friendsofsymfony/rest-bundle": "~1.7",
         "eko/feedbundle": "^1.2.7",
         "lunetics/locale-bundle": "^2.5",
         "burgov/key-value-form-bundle": "^1.4",
-        "egeloen/ckeditor-bundle": "^4.0"
+        "egeloen/ckeditor-bundle": "^4.0",
+        "symfony-cmf/sonata-admin-integration-bundle": "^1.0",
+        "symfony-cmf/routing-bundle": "^2.0",
+        "symfony-cmf/menu-bundle": "^2.0",
+        "symfony-cmf/block-bundle": "^2.0",
+        "symfony-cmf/core-bundle": "^2.0",
+        "symfony-cmf/content-bundle": "^2.0"
```

Comment:
All these bundles are required in `symfony-cmf/symfony-cmf`, so they can be removed.